src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -160,6 +163,9 @@ private void verify(Configuration config, String filePath, String[] expected,
             final IndentComment... linesWithWarn) throws Exception {
         final Checker checker = createChecker(config);
         checker.addListener(new IndentAudit(linesWithWarn));
+        checker.setLocaleCountry(Locale.ROOT.getCountry());
+        checker.setLocaleLanguage(Locale.ROOT.getLanguage());
+        checker.finishLocalSetup();
```

Comment:
I will split this to separate issue.

@rnveach , please approve PR . I am ready to merge.